src/cljc/rems/common/roles.cljc

```diff
@@ -32,7 +33,29 @@
             (boolean (some (set roles) context/*roles*)))
      :cljs (some (set roles) (:roles @(rf/subscribe [:identity])))))
 
+(defn is-with-organization?
```

Comment:
We could name this `is-owner-of-organization?' or so because I think organization membership is a related but different concept. We want this to refer specifically to ownership.